x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/tree/EsqlNodeSubclassTests.java

```diff
@@ -394,6 +395,24 @@ public void accept(Page page) {
         }
         Class<?> argClass = (Class<?>) argType;
 
+        // Handle array types - can't mock arrays, so create them directly
```

Comment:
I just realized that our query plans never really use arrays, but generally lists.

consistency nit: any reason to use arrays in `LookupMergeDropExec` (which necessitates this test change)? (Except that it's admittedly weird to not use `int[]`.)